src/ui/FieldTable/FieldValue.ts

```diff
@@ -69,13 +70,18 @@ export class FieldValue extends Component {
      */
     htmlValue: ComponentOptions.buildBooleanOption({ defaultValue: false }),
     /**
-     * Specifies if the field value is to be split at each semicolon (<code>;</code>).
+     * Specifies if the field value is to be split at each separator.
      * This is useful for splitting groups by a facet field.<br/>
      * The values displayed are split by commas (<code>,</code>).<br/>
      * The default value is <code>false</code>.
      */
     splitValues: ComponentOptions.buildBooleanOption({ defaultValue: false }),
     /**
+     * Specifies the string used to split multi value fields.

```

Comment:
Please add at least one UT for the options that checks it's default value + the possibility to modify it.